server/src/main/java/org/elasticsearch/cluster/ClusterInfo.java

```diff
@@ -40,10 +40,10 @@ public class ClusterInfo implements ToXContentFragment, Writeable {
 
     private final Map<String, DiskUsage> leastAvailableSpaceUsage;
     private final Map<String, DiskUsage> mostAvailableSpaceUsage;
-    final Map<String, Long> shardSizes;
-    final Map<ShardId, Long> shardDataSetSizes;
-    final Map<ShardRouting, String> routingToDataPath;
-    final Map<NodeAndPath, ReservedSpace> reservedSpace;
+    public final Map<String, Long> shardSizes;
+    public final Map<ShardId, Long> shardDataSetSizes;
+    public final Map<ShardRouting, String> routingToDataPath;
+    public final Map<NodeAndPath, ReservedSpace> reservedSpace;
```

Comment:
I'd be tempted to move `ClusterInfoSimulator` into `org.elasticsearch.cluster` rather than expose these to all packages.

Reply:
This was my original intention as well, but I wanted to avoid introducing something in `org.elasticsearch.cluster`. May be ClusterInfo and ClusterInfoService should actually be renamed or in some sub-package, but that is a question for another PR.